src/dune_rules/merlin/merlin.ml

```diff
@@ -224,6 +244,7 @@ module Processed = struct
       printf "STDLIB %s\n" (serialize_path stdlib_dir));
     Path.Set.iter obj_dirs ~f:(fun p -> printf "B %s\n" (serialize_path p));
     Path.Set.iter src_dirs ~f:(fun p -> printf "S %s\n" (serialize_path p));
+    List.iter indexes ~f:(fun p -> printf "INDEX %s\n" (serialize_path p));
```

Comment:
What would happen to older versions of merlin if they'd encounter this directive?

Reply:
Since Merlin 4.6 unknown configuration tags are ignored when `dune ocaml-merlin` is used as a configuration reader, so it should not trigger any error.